File: TriathlonServerJava/Common/src/main/java/ro/mpp/triathlon/model/Participant.java
package ro.mpp.triathlon.model;

//import model.Entity;

public class Participant extends Entity {
    private String name;
    private Integer totalPoints;

    public Participant(int id, String name, Integer totalPoints) {
        super(id);
        this.name = name;
        this.totalPoints = totalPoints;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getTotalPoints() {
        return totalPoints;
    }

    public void setTotalPoints(Integer totalPoints) {
        this.totalPoints = totalPoints;
    }
}

File: TriathlonCrossPlatform/Traithlon.Model/Participant.cs
namespace Triathlon.Model
{
    public class Participant : Entity
    {
        public new int Id { get => base.Id; set => base.Id = value; }

        public string Name { get; set; }
        public int TotalPoints { get; set; }

        public Participant(int id, string name, int totalPoints) : base(id)
        {
            this.Id = id;
            this.Name = name;
            this.TotalPoints = totalPoints;
        }
    }
}